react-paw-mailmerge/src/Helpers/CsvFunctions.tsx

```diff
@@ -15,35 +24,59 @@ export function ParseCsv(myFile:File, options:CsvOptions, resolve: (s: Array<obj
         dynamicTyping: true, // dynamically type, so that numbers are numbers and bool are bools and not strings
         keepEmptyRows: false, 
         complete: function(results:ParseResult<object>, file:File) { // This is what is called when we successfully parse
-            resolve(NormalizeJsonObjectResult(results, options.dataHasHeader));
+            resolve(NormalizeJsonObjectResult(results, options));
         },
         error:  function(results:object, file:File) { // This is what is called when we fail to parse
             console.error("Errors:", results, file);
             reject("There was an error", results);
         },
-        header: options.dataHasHeader,
+        header: options.header,
         skipEmptyLines: true
     } as ParseLocalConfig<unknown, File>;
 
     parse(myFile, config);
   }
 
-function NormalizeJsonObjectResult(results:ParseResult<object>, hasHeader:boolean) : Array<object>
+function HandleNullValues(val: any, nullFieldOption : NullFieldOptionEnum) : any
+{
+    if(val === null){
```

Comment:
The `if` block might not be necessary. Since null values in csv files are represented by consecutive delimiters, I don't think `val` will ever equal `null`. A null value will already be represented as an empty string by default.